tests/unittest.py

```diff
@@ -170,6 +169,7 @@ def create_peer_from_builder(self, builder, start_manager=True):
 
         if start_manager:
             manager.start()
+            self.clock.running = True
```

Comment:
@msbrogli this will probably be a relevant line to change in https://github.com/HathorNetwork/hathor-core/pull/584

Reply:
Referenced in https://github.com/HathorNetwork/hathor-core/pull/584#issuecomment-1632904296